articles/azure/azs-how-resize-os-drive.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: How to resize an unmanaged disk - PowerShell | UKCloud Ltd
-description: Provides help for resizing an unmanaged disk on UKCloud for Microsoft Azure
+title: How to resize an OS drive | UKCloud Ltd
```

Comment:
title metadata tag should match the main article heading, so change this to

How to resize an OS drive using PowerShell

Reply:
Fixed